biit_server/meeting.py

```diff
@@ -48,14 +48,20 @@ def __init__(
         self.location = location
         self.meeting_type = meeting_type
 
-    def add_user(self, user) -> List[str]:
-        self.user_list.append(user)
+    def add_user(self, user) -> Dict[str, bool]:
+        self.user_list[user] = None
```

Comment:
does this get updated in the DB automatically?

Reply:
This gets updated in the db when meeting_db.update is called